### PR TITLE
OCM-13016 | fix: Include path in create acc/op roles manual mode

### PR DESCRIPTION
**Pull request by hunterkepley**

### CODE REVIEW

File: pkg/roles/utils.go [hunk 1]
```diff
@@ -36,10 +36,11 @@ func GetHcpSharedVpcPolicyDetails(r *rosa.Runtime, roleArn string) (bool, string
 	if err != nil {
 		return false, "", "", err
 	}
+	path, err := aws.GetPathFromARN(roleArn)
```
Comment (marcolan018) [inline pkg/roles/utils.go:39]: the "err" is ignored?

Comment (hunterkepley) [reply pkg/roles/utils.go:39]: oops thanks

**Pull request merged**